.travis.yml

```diff
@@ -4,31 +4,29 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
-env:
-  - SYMFONY_VERSION="2.7.*,>2.7.6"
-
 sudo: false
 
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 matrix:
   include:
     - php: 5.3
       env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
-    - php: 5.6
-      env: SYMFONY_VERSION=2.3.* SYMFONY_DEPRECATIONS_HELPER=weak
-    - php: 5.6
+    - php: 7.0
+      env: SYMFONY_VERSION=2.7.*
+    - php: 7.0
       env: SYMFONY_VERSION=2.8.*
-    - php: 5.6
-      env: SYMFONY_VERSION=3.0.*
-  allow_failures:
-    - php: 5.6
-      env: SYMFONY_VERSION=3.0.*
+  fast_finish: true
 
 before_install:
   - composer self-update || true
-  - sh -c 'if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi;'
-  - composer require --no-update symfony/symfony:${SYMFONY_VERSION}
+  - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/symfony=$SYMFONY_VERSION; fi;
```

Comment:
for some reason it then picks 2.6.5 if we set no SYMFONY_VERSION:
https://travis-ci.org/doctrine/DoctrinePHPCRBundle/jobs/95231164#L143